createProductCard = (product) => {
    try {
    }
    catch(err){ console.log("createProductCard: ERROR", err, product);}
}

loadCategoryProducts = async (categoryId) => {
    try {
    }
    catch(err){ console.log("loadCategoryProducts: ERROR", err, categoryId);}
}

